telemetry_core/src/test/java/com/newrelic/telemetry/transport/BatchDataSenderTest.java

```diff
@@ -65,4 +69,31 @@ void testSecondaryUserAgent() throws Exception {
 
     assertEquals(new Response(202, "OK", "yepyep"), response);
   }
+
+  @Test
+  void testCapturingCaseOfIoException() throws Exception {
+    URL endpointURl = new URL("http://foo.com");
```

Comment:
Let's try and standardize on `example.com` for urls.

Reply:
I'll update all the tests in this file for that.

Reply:
done